check_signed_off.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+echo "Checking range: ${TRAVIS_COMMIT_RANGE}"
+
+NEW_COMMITS=( $(git log  --format=format:%H "$TRAVIS_COMMIT_RANGE") )
+
+for i in "${NEW_COMMITS[@]}"
+do
+    PARENTS=$(git cat-file -p "$i" | grep -c "parent")
+    if [ "$PARENTS" -gt 1 ];
+     then
+        #if the commit is a merge, dont check the commit msg.
+        continue
+     else
+        COMMIT_MSG=$(git show -s --format=%B "$i")
+        echo "$COMMIT_MSG" | grep '^Signed-off-by: ' >/dev/null || {
```

Comment:
We need to be strict here and check that the *author* is signing it off, so the value here should be compared against `git show -s --format="%an <%ae>"` of the commit.